RecipesLyricsMashup/RecipesLyricsMashupV2/RecipeLyricsMashup/RecipeLyricsMashup/Pages/Index.cshtml.cs

```diff
@@ -19,23 +19,26 @@ public IndexModel(ILogger<IndexModel> logger)
         {
             _logger = logger;
         }
+        [BindProperty(SupportsGet = true)]
+        public string SearchString { get; set; } //Morgan added
 
         public IActionResult OnGet(string search)
         {
             Encoding ascii = Encoding.ASCII;
             //string search = HttpUtility.HtmlEncode(sort);
-            Console.WriteLine(search);
+            //Console.WriteLine(search);
             string recipeEndpoint = "http://www.recipepuppy.com/api/?q=" + search;
             string recipeJson = GetData(recipeEndpoint);
             QuickTypeRecipe.Recipe recipeResult = QuickTypeRecipe.Recipe.FromJson(recipeJson);
             QuickTypeRecipe.Result[] recipes = recipeResult.Results;
-            ViewData["recipePuppyJson"] = recipes;
+            ViewData["recipePuppyRecipies"] = recipes;
             string accessToken = System.IO.File.ReadAllText("APIToken.txt");
             string geniusEndpoint = "https://api.genius.com/search?q="+ search + "&access_token=" + accessToken;
             string geniusResultsJson = GetData(geniusEndpoint);
             QuickType.SearchResult searchResults = QuickType.SearchResult.FromJson(geniusResultsJson);
             QuickType.Hit[] results = searchResults.Response.Hits;
             ViewData["geniusResults"] = results;
+            ViewData["SearchString"] = search; //Morgan Added

```

Comment:
This will pass the SearchString that was bound to the search box back to the front end upon loading the page.